src/utils/azure.ts

```diff
@@ -162,3 +166,29 @@ export const hasCosmosConnection = async (
     return false;
   }
 };
+
+export interface IServicePrincipalCreds {
+  readonly clientId: string;
+  readonly secret: string;
+  readonly tenantId: string;
+}
+
+export function getGraphRbacManagementClient(
+  adb2cCreds: IServicePrincipalCreds
+): TaskEither<Error, GraphRbacManagementClient> {
+  return tryCatch(
+    () =>
+      msRestNodeAuth.loginWithServicePrincipalSecret(
```

Comment:
maybe `loginWithUserAndPassword` or with az client login ?

Reply:
Added in [b920954](https://github.com/pagopa/io-ops/pull/128/commits/b920954548320f3fb5b39bc7e52b4e06910be43a)